test/IntegrationTests/HttpTests.cs

```diff
@@ -37,7 +37,11 @@ public void SubmitTraces(string propagators)
             return true;
         });
         Span? serverSpan = null;
+#if NET7_0_OR_GREATER
```

Comment:
This test does not need an update. Removing this update will fix CI test failures. 

This kind of test update is needed, if we manually upgrade instrumentation libraries for .NET 7.0/8.0.